even_fib_nums.js

```diff
@@ -5,31 +5,50 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
+function _sumFibs(maxFibValue) {
   var sum = 0;
-
-  // do your work here
+console.log(maxFibValue)
+  var arr = [1, 2];
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
Is there a way to merge the two for loops together so that there's only one loop that can handle doing both generating the next fibonacci number while keeping track of even fibonacci numbers and summing them?